gist.go

```diff
@@ -0,0 +1,92 @@
+package model
+
+// Typed constants. Types are not exported as you can still export the values 
+// for use in `if` and `switch` statements.
+type channelType string
```

Comment:
Just do we don't lose track
https://github.com/mattermost/mattermost-server/pull/14301#discussion_r411439453